package.json

```
@@ -8,7 +8,7 @@
     "watch:eleventy": "eleventy --serve",
     "build:sass": "sass  --no-source-map src/sass:public/css",
     "build:eleventy": "eleventy",
-    "postbuild": "lightningcss --minify --targets '> 0.25%, not IE 11' public/css/*.css -o public/css/*.css",
+    "postbuild": "BROWSERSLIST_CONFIG='./package.json' lightningcss --browserslist --minify public/css/*.css -o public/css/*.css",
```

It looks like it will use the `browserslist` as default, I think when I tested after initially implementing this change it didn't seem to be working that way but now it is (also backed-up by the docs you pointed to).

```suggestion
    "postbuild": "lightningcss --browserslist --minify public/css/*.css -o public/css/*.css",
```